### DISCUSSION TITLE
Multiple target tracking?

### DISCUSSION THREAD

**Discussion by MattW87** [Stellarium/stellarium#3493] [Q&A]
Hi all,

I'm very new to Stellarium and I was wondering if there was a way to track multiple objects, or at least to highlight multiple objects for tracking purposes?

Thanks and Kind Regards
Matt W

**Comment by gzotti** [comment]
The AstroCalc panel may help. See User Guide, 4.6.2.

**Reply by MattW87** [reply]
thanks for the quick reply!  I'll have a play around and see what it can do for me.

**Comment by alex-w** [comment]
Observing Listst can help you
